### PR TITLE
feat(docker-camunda-bpm-platform): Added sql and postgresql version arguements

### PR DESCRIPTION
**Pull request by punitdarira**
related to https://github.com/camunda/camunda-bpm-platform/issues/2692

### CODE REVIEW

**Comment by punitdarira** [comment]
https://github.com/camunda/docker-camunda-bpm-platform/pull/295

**Comment by danielkelemen** [comment]
👍 Looks good to me.
🔧 Could you also add this new build-arg to the README so people can find it? I'd say a [new sub-section here](https://github.com/camunda/docker-camunda-bpm-platform/blob/next/README.md?plain=1#L398) about how to build with custom db versions with an example cli would be nice.

I'll test your changes with a local branch in the meantime. https://github.com/camunda/docker-camunda-bpm-platform/pull/298

**Comment by punitdarira** [comment]
> 👍 Looks good to me. 🔧 Could you also add this new build-arg to the README so people can find it? I'd say a [new sub-section here](https://github.com/camunda/docker-camunda-bpm-platform/blob/next/README.md?plain=1#L398) about how to build with custom db versions with an example cli would be nice.
> 
> I'll test your changes with a local branch in the meantime. #298

Sure @danielkelemen

File: README.md [hunk 1]
```diff
@@ -396,6 +396,17 @@ docker build -t camunda-bpm-platform \
   --build-arg MAVEN_PROXY_PASSWORD=${PROXY_PASSWORD} \
   .
 ```
+### Override MySQL and PostgreSQL driver versions. 
+By default, the driver versions are fetched from github.com/camunda/camunda-bpm-platform/blob/master/database/pom.xml. That can be overriden by passing `MYSQL_VERSION` and `POSTGRESQL_VERSION` build args
```
Comment (danielkelemen) [inline README.md]: Thank you, changes look good!
🔧 Let's make this a full clickable link.

```suggestion
By default, the driver versions are fetched from https://github.com/camunda/camunda-bpm-platform/blob/master/database/pom.xml. That can be overriden by passing `MYSQL_VERSION` and `POSTGRESQL_VERSION` build args
```

**Review by danielkelemen — APPROVED** [review]
👍 Looks good!

**Pull request merged**